tests/tracee_test.go

```diff
@@ -126,3 +129,92 @@ func TestWebhookIntegration(t *testing.T) {
 	t.Log("Terminating the Tracee container...")
 	assert.NoError(t, exec.Command("docker", "kill", containerID).Run())
 }
+
+type traceeContainer struct {
```

Comment:
Do we need this type? I don't see any methods defined on traceeContainer.

Reply:
Yes AFAIK because Container is an interface and something needs to implement it https://github.com/testcontainers/testcontainers-go/blob/master/container.go#L34

Is there a way to call functions of an interface without a struct implementing them?